### ISSUE TITLE
Deploy with Llama2

### ISSUE THREAD

**Issue by Libresse** [AI-Citizen/SolidGPT#76]
Hi SolidGPT team, 

Are there any ways I can deploy your product with Llama2 or any other open source model?

I saw you have [Llama manager](https://github.com/AI-Citizen/SolidGPT/blob/main/solidgpt/src/manager/llamanager.py) but didn't find a way to use it. 

BR,

**Comment by wwwwww1020** [comment]
Yes, we are working on using opensource model Llama2 as the LLM core to implement fully local deployment. We will update more about this cool feature in the future. Btw, we will publish our roadmap real soon

**Comment by Libresse** [comment]
@wwwwww1020 Does the SolidGPT fully support Azure integration, any best practice for that?

**Comment by kodiii** [comment]
would be great if was possible to use with LM Studio app

**Comment by wwwwww1020** [comment]
Yes you can easily deploy the project into Azure with Static App , container app and blob storage. We have provide the blob storage client manager let user easily store their data into the blob. We will provide more context about deployment in the further

**Comment by wwwwww1020** [comment]
> would be great if was possible to use with LM Studio app

good idea we will do some research on this

**Comment by Libresse** [comment]
So are you going to support Chinese LLM like ERNIE in the future?

**Comment by wwwwww1020** [comment]
> So are you going to support Chinese LLM like ERNIE in the future?

Yes we are going to adding more model like ERNIE

**Comment by bhupesh-sf** [comment]
Hey, did we have any update on LM Studio or anything similar which gives openAI compatible API?
